src/cli/cli_coap.cpp

```diff
@@ -796,27 +796,18 @@ otError Coap::ProcessRequest(Arg aArgs[], otCoapCode aCoapCode)
 
 otError Coap::Process(Arg aArgs[])
 {
-#define CmdEntry(aCommandString)                            \
-    {                                                       \
-        aCommandString, &Coap::Process<Cmd(aCommandString)> \
-    }
+#define CmdEntry(aCommandString) {aCommandString, &Coap::Process<Cmd(aCommandString)>}
 
     static constexpr Command kCommands[] = {
 #if OPENTHREAD_CONFIG_COAP_OBSERVE_API_ENABLE
         CmdEntry("cancel"),
 #endif
-        CmdEntry("delete"),
-        CmdEntry("get"),
+        CmdEntry("delete"),     CmdEntry("get"),
 #if OPENTHREAD_CONFIG_COAP_OBSERVE_API_ENABLE
         CmdEntry("observe"),
 #endif
-        CmdEntry("parameters"),
-        CmdEntry("post"),
-        CmdEntry("put"),
-        CmdEntry("resource"),
-        CmdEntry("set"),
-        CmdEntry("start"),
-        CmdEntry("stop"),
+        CmdEntry("parameters"), CmdEntry("post"),  CmdEntry("put"),  CmdEntry("resource"),
+        CmdEntry("set"),        CmdEntry("start"), CmdEntry("stop"),
```

Comment:
Previously this was the change which prevented me from upgrading clang-format. I personally prefer the one entry per line style.

Reply:
I would suggest we  keep it as `clang-format` want (not use `// clang-format off/on` to turn it off).

I think the good thing is it is would be consistent (previously some arrays were one per line, some arranged differently - at least I could never tell the reason).

Reply:
Sounds good to me. Thanks!